### DISCUSSION TITLE
Battlenet issues

### DISCUSSION THREAD

**Discussion by Thevoid2gaming** [moraroy/NonSteamLaunchers-On-Steam-Deck#546] [General]
Here is the log for my installs
[NonSteamLaunchers-install.log](https://github.com/user-attachments/files/18267375/NonSteamLaunchers-install.log)

**Comment by moraroy** [comment]
Everything looks correct, so if I'm understanding correctly it simply won't run?

**Reply by Thevoid2gaming** [reply]
I can install battle net and the setup for battle net will appear but after everything is done it just doesn’t appear anywhere so I can’t access it.

**Reply by moraroy** [reply]
@Thevoid2gaming oh OK gotcha, did you stop and restart the scanner?

**Reply by moraroy** [reply]
@Thevoid2gaming just so you know according to your log, it says that the shortcut exists in your library. It's possible it may have added to another user's library, do you have multiple accounts?

**Reply by Thevoid2gaming** [reply]
I’m not sure what that means. What is the scanner?

**Reply by Thevoid2gaming** [reply]
I also don’t have any other accounts

**Reply by moraroy** [reply]
Are you using the desktop version of NSL or the Decky Plug-in Version?

**Reply by Thevoid2gaming** [reply]
Desktop version

**Reply by moraroy** [reply]
@Thevoid2gaming open up the NonSteamLaunchers.desktop again and press the "Stop Scanner Button" and reset it. Then look in your library and see if the battlenet is there.

**Reply by Thevoid2gaming** [reply]
It did not appear after restarting the scanner

**Reply by moraroy** [reply]
@Thevoid2gaming show me that current log, because your old log says it exists.

**Reply by Thevoid2gaming** [reply]
[NonSteamLaunchers-install.log](https://github.com/user-attachments/files/18308183/NonSteamLaunchers-install.log)

**Reply by moraroy** [reply]
@Thevoid2gaming strange.... Existing Shortcuts: AppID for Tomb Raider GOTY: -928394101 AppID for Epic Games: -456079619 AppID for Gog Galaxy: -66626 AppID for Ubisoft Connect: -231869725 AppID for Battle.net: -508946163 AppID for EA App: -846878561  , is there any way it's been accidentally hidden in your library? Maybe a filter is on, it should be in your non steam section newly added

**Reply by Thevoid2gaming** [reply]
Omg it’s for some reason counted as hidden. Thank you so much for your help!

**Reply by moraroy** [reply]
@Thevoid2gaming wow no problem! I'm glad it's all working now!
